tests/acceptance/features/apiSharingNg1/listPermissions.feature

```diff
@@ -2307,3 +2272,198 @@ Feature: List a sharing permissions
           }
       }
       """
+
+  @env-config
+  Scenario: user lists permissions of a folder in personal space
+    Given user "Alice" has created folder "folder"
+    And the administrator has enabled the permissions role "Secure Viewer"
+    When user "Alice" gets permissions list for folder "folder" of the space "Personal" using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "@libre.graph.permissions.actions.allowedValues",
+          "@libre.graph.permissions.roles.allowedValues"
+        ],
+        "properties": {
+          "@libre.graph.permissions.actions.allowedValues": {
+            "type": "array",
+            "enum": [
+              [
+                "libre.graph/driveItem/permissions/create",
+                "libre.graph/driveItem/children/create",
+                "libre.graph/driveItem/standard/delete",
+                "libre.graph/driveItem/path/read",
+                "libre.graph/driveItem/quota/read",
```

Comment:
if this is same with default roles then we can skip this check - just check that the property is there